doc/content/getting-started/troubleshooting/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Troubleshooting"
+description: ""
+---
+
+This section provides help for common issues and frequently asked questions you may have when getting started with {{% tts %}}. If you encounter a bug, please [file an issue](https://github.com/TheThingsNetwork/lorawan-stack/issues/new/choose). If you have questions not covered here, please ask in [{{% tts %}} Forum](https://www.thethingsnetwork.org/forum/c/network-and-routing/v3).
+
+<!--more-->
+
+## My gateway won't connect. What do I do?
+
+- Double check your gateway settings in {{% tts %}}. Ensure your Gateway EUI is correct
+- Double check the configuration settings on your gateway. Is the network address the same address you use to connect to {{% tts %}}? Are your ports correct?
+- If using CUPS, double check your API Keys and certificates
+
+## My device won't join. What do I do?
+
+- Double check your Device EUI, Join EUI, LoRaWAN Version, AppKey, and NwkKey
+- Check gateway and application events for traffic from your device
+- Double check frequency plan settings in your end device and gateways (they must be matching)
+- Double check your network connection. If there is a slow connection from the server to the gateway, the join accept message may be sent too late (this can happen when a gateway uses 3G as a backhaul)
+- Check for an overused App Nonce
+
+## No downlinks are reaching my device. What do I do?
+
+- Check gateway and application events for traffic from your device
+- Check duty cycle restrictions
+- Device clock drift often occurs when SF12 is used
+- Check your antenna connections
+
+## {{% tts %}} is no longer receiving uplinks from my device. What do I do?
+
+- Check gateway and application events for traffic from your device
+- Check for FCnt mismatch on ABP devices
+
+## How do I get gateway logs?
```

Comment:
I.e. here. Note that these are not logs, but events

Gateway logs are on the gateway